test/functional/users_controller_test.rb

```diff
@@ -2,83 +2,65 @@
 
 class UsersControllerTest < ActionController::TestCase
   setup do
-    @user  = users(:kate)
-    @admin = users(:nixon)
   end
 
   test 'should not get index as user' do
-    get :index, {},  user_id: @user
+    kate = sign_in('user', users(:kate))
+    get :index, {},  user_id: kate
     assert_response :forbidden
   end
 
   test 'should get index as admin' do
-    get :index, {},  user_id: @admin
+    nixon = sign_in('user', users(:nixon))
+    get :index, {},  user_id: nixon
     assert_response :success
     assert_not_nil assigns(:users)
     assert_valid_html response.body
   end
 
   test 'should get index paged' do
-    get :index, { format: 'json', per_page: 1 },  user_id: @admin
+    nixon = sign_in('user', users(:nixon))
+    get :index, { format: 'json', per_page: 1 },  user_id: nixon
     assert_response :success
     assert JSON.parse(response.body).count == 1, 'Should only have got one user back'
   end
 
   test 'should get index sorted' do
-    get :index, { format: 'json', sort: 'ASC' },  user_id: @admin
+    nixon = sign_in('user', users(:nixon))
+    get :index, { format: 'json', sort: 'ASC' },  user_id: nixon
     assert_response :success
     body = JSON.parse(response.body)
     assert Date.parse(body[0]['createdAt']) < Date.parse(body[1]['createdAt']), 'Was not in ascending order'
   end
 
   test 'should get index searched' do
-    get :index, { format: 'json', search: 'kate', sort: 'ASC' },  user_id: @admin
+    nixon = sign_in('user', users(:nixon))
+    get :index, { format: 'json', search: 'kate', sort: 'ASC' },  user_id: nixon
     assert_response :success
   end
 
   test 'should get new' do
-    get :new
-    assert_response :success
-    assert_valid_html response.body
+    skip('No longer valid as of devise integration')
   end
 
   test 'should create user' do
-    assert_difference('User.count') do
-      post :create, user: { email: 'john@example.com', name: 'John F.',
-                            password: 'iguana', password_confirmation: 'iguana' }
-      puts flash[:debug] unless flash[:debug].nil?
-    end
-
-    assert_redirected_to user_path(assigns(:user))
-
-    john = User.find_by_email('john@example.com')
-
-    assert_not_nil john
-    assert_equal john.validated?, false,
-      'New user should not be validated'
+    skip('No longer valid as of devise integration')
```

Comment:
Yes, I just left it like that so we know what tests are being removed and no one thinks we are just removing a ton of tests. I am planning on removing all the skipped tests in another pull request so it is clear what ones are being removed and why.